src/config-utils.ts

```
@@ -1168,7 +1168,8 @@ export function parsePacksFromConfig(
       throw new Error(getPacksInvalid(configFile));
     }
     if (!languages.includes(lang as Language)) {
-      throw new Error(getPacksRequireLanguage(lang, configFile));
+      // This particular language is not being analyzed in this run.
```

I suggest a log message at least.

---

Done. Needed to pass the logger in from up the call chain.

Also, added a case where we actually do want to throw an error.

---

Would it make sense to have an integration test that resembles the customer setup you observed?

---

Easiest thing to do is to augment the existing packaging tests to include other unused languages. I don't think we have any integration tests that are meant to fail (so we don't have the infrastructure check that an invalid language does indeed fail the analysis).